### ISSUE TITLE
CUERipper feature request: set Artist value for multiple tracks

### ISSUE THREAD

**Issue by tjanas** [gchudov/cuetools.net#134]
For a Various Artists disc where the artists varies between tracks, it can become tedious to assign a specific artist to multiple tracks. It would be nice if there was a way to highlight / select multiple tracks and then assign a specific artists to those selected tracks. Example use-case (where I am replacing Japanese artist text with English text):
![image](https://user-images.githubusercontent.com/20304619/137340781-b24befb4-5f9f-4161-b76f-0b6912d5abd8.png)

**Comment by RDamman** [comment]
Yes, I was also thinking about a nice feature like that. Maybe it could be implemented by copy/paste to multiple selected fields at once.

**Comment by RDamman** [comment]
Implemented in my test version: [Releases](https://github.com/RDamman/cuetools.net/releases)
See also: [Pull request](https://github.com/gchudov/cuetools.net/pull/391)
